pages/work/jebbit.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div class="section flex flex-wrap justify-center">
+    <div class="max-w-2xl w-full">
+      <div class="p-4 lg:p-12">
+        <div class="text-xs">
+          Case Study
+        </div>
+
+        <h1 class="text-4xl lg:text-6xl">
+          Jebbit
+        </h1>
+
+        <p>
+          Jebbit is an enterprise software platform that enables a future of
+          total data transparency between brands and consumers. Jebbit’s mission
+          is to make every brand-and-consumer interaction assumption free by
+          asking consumers explicitly and directly about their interests,
+          motivations, and preferences.
+        </p>
+      </div>
+
+      <div class="bg-black flex flex-wrap h-64 md:h-140 justify-center items-center">
+        <div class="w-1/2">
+          <JebbitLogo class="fill-white h-auto w-full" />
+        </div>
+      </div>
+
+      <div class="p-4 lg:p-12">
+        <h4>Challenge</h4>
+
+        <p>
+          Jebbit had several years of user feedback and data on the difficulty
+          in using the Campaign Builder product and sought to create a
+          lightweight version that is more accessible and easier to get started
```

Comment:
The simple builder is also mobile friendly right? Perhaps we should talk more about the challenge being that anyone anywhere, on any device, can more easily create these campaigns and that the process is also now more intuitive.